src/Queries.re

```diff
@@ -102,6 +102,22 @@ module AllCommunitiesQueryConfig = [%graphql
 module AllCommunitiesQuery =
   ReasonApolloHooks.Query.Make(AllCommunitiesQueryConfig);
 
+module CommunitySettingsQueryConfig = [%graphql
+  {|
+    query communitySettings($communityName: String!) {
+      community_settings(where: {community: {name: {_eq: $communityName }}})
```

Comment:
I think hasura supports `limit 1` for such cases, will it make it understand that the return type is not an array but one entry?

Reply:
Added `limit:1`. It still returns an array though, so nothing else has changed.

Reply:
Hm.. so there is really no way to tell the type system that it is going to be only one result?

Reply:
I don't think so - not with the standard Hasura way at least. A custom Postgres function exposed via Hasura might work, but not worth it in this case.